docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -27,7 +27,7 @@ weight: 100
 
 # Table
 
-Tables are very flexible, supporting multiple modes for time series and for tables, annotation, and raw JSON data. This visualization also provides date formatting, value formatting, and coloring options.
+Tables are very flexible, supporting multiple modes for time series and for tables, annotation, and raw JSON data. This visualization also provides date formatting, value formatting, and coloring options. In addition to formatting and coloring options, Grafana also provides a variety of _Cell types_ which can be used to display gauges, sparklines, and other rich data displays.
```

Comment:
We want to avoid passive voice as it's less clear than active voice. I've made the same type of change to the other additions.

```suggestion
Tables are very flexible, supporting multiple modes for time series and for tables, annotation, and raw JSON data. This visualization also provides date formatting, value formatting, and coloring options. In addition to formatting and coloring options, Grafana also provides a variety of _Cell types_ which you can use to display gauges, sparklines, and other rich data displays.
```